org.eclipse.virgo.bundlor/src/test/java/org/eclipse/virgo/bundlor/support/contributors/xml/SpringApplicationContextArtifactAnalyzerTests.java

```diff
@@ -28,6 +29,8 @@ public class SpringApplicationContextArtifactAnalyzerTests {
     private final SpringApplicationContextArtifactAnalyzer analyzer = new SpringApplicationContextArtifactAnalyzer();
 
     @Test
+    @Ignore
```

Comment:
I don't think the issue is related to github actions, the tests fail for me even in master when I build locally on my computer.

Reply:
Both tests fail due to:

JAXP0801002: the compiler encountered an XPath expression containing '101' operators that exceeds the '100' limit set by 'FEATURE_SECURE_PROCESSING'.

which seems to be solved by setting certain system properties in the build to extend default limits introduced in more recent JDK 8 builds:

https://www.oracle.com/java/technologies/javase/8u331-relnotes.html